build/scraper.js

```diff
@@ -79,6 +79,14 @@ class Scraper {
       const raw = await getJSON(`https://content.warframe.com/PublicExport/Manifest/${endpoint}`);
       const data = raw ? raw[`Export${category}`] : undefined;
       bar.tick();
+
+      if (category === 'Upgrades') {
+        const modSets = raw.ExportModSet.map((ms) => ({
+          ...ms,
```

Comment:
It's bad practice to use shortened names when there's no need to, even if it's clear what they should mean.
```suggestion
        const modSets = raw.ExportModSet.map((modSet) => ({
          ...modSet,
```
kudos to you, for not modifying the original modSet object.